addons/armaos/functions/fnc_computer_addActionStandby.sqf

```diff
@@ -0,0 +1,31 @@
+/**
+ * Adds the "standby" ACE interaction for a given computer object. Computer texture changes accordingly.
+ *
+ * Arguments:
+ * 1: Computer <OBJECT>
+ *
+ * Results:
+ * None
+ */
+
+params ["_computer"];
+
+_standbyTime = 3;
```

Comment:
private

Reply:
changed in commit https://github.com/y0014984/Advanced-Equipment/pull/90/commits/351ca2f7548ca9717ff26f606317f2c7e56c0763